admin/timescale/create_tables.sql

```diff
@@ -202,10 +202,23 @@ CREATE TABLE similarity.recording (
     metadata JSONB NOT NULL
 );
 
+CREATE TABLE similarity.recording_prod (
```

Comment:
One set of tables is for production use and another is for use continuing development, yes? If that is right I think the development tables should have a _dev postfix and the _prod tables should not have a suffix. Having production tables without suffixes makes those tables consistent with the other production tables that do not have _prod appending. What do you think?

Reply:
Yes. Renaming makes sense, will do.